google-http-client/src/main/java/com/google/api/client/http/UrlEncodedContent.java

```diff
@@ -42,18 +42,33 @@
  *
  * <p>Implementation is not thread-safe.
  *
- * @since 1.0
  * @author Yaniv Inbar
+ * @since 1.0
  */
 public class UrlEncodedContent extends AbstractHttpContent {
 
   /** Key name/value data. */
   private Object data;
 
+  /** Use URI Path encoder flag. False by default (use legacy and deprecated escapeUri) */
+  private Boolean useEscapeUriPathEncoding;
```

Comment:
rename this please to something shorter without "use"

Reply:
I tried a new name. Let me know what do you think.